src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -178,21 +182,21 @@ final class ScalaCheckFramework extends Framework {
 
   val name = "ScalaCheck"
 
-  def fingerprints: Array[Fingerprint] = Array(
+  def fingerprints: Array[Fingerprint | Null] = Array(
     mkFP(false, "org.scalacheck.Properties"),
     mkFP(false, "org.scalacheck.Prop"),
     mkFP(true, "org.scalacheck.Properties"),
     mkFP(true, "org.scalacheck.Prop")
   )
 
-  def runner(_args: Array[String], _remoteArgs: Array[String],
-    _loader: ClassLoader
+  def runner(_args: Array[String | Null] | Null, _remoteArgs: Array[String | Null] | Null,
```

Comment:
A Scala method overrides some Java method. We need to add |Null to the method's parameter types to make the override still work.